### PR TITLE
Added overflow-y: auto; to slide-pane

### LINKED ISSUE #641: Content which does not fit into the slide-pane is cut out and cannot be reached.
**Enhancement**

When I am using the slide-pane widget with contents bigger than the slide-pane size I want to be able to scroll down so that I can see each parts of the slide-pane's contents.

I think https://github.com/dojo/widgets/issues/454 is similar to this one.

I also discussed certain issues on discord, where @agubler helped me to apply different styles to links when the slide-pane opens.

Package Version: 4.0.2-pre

**Code**

When you put more lorem ipsum contents into the example code of the slide-pane (See https://github.com/SimonScholz/widgets/commit/e4b185e6b1fc0e035483930d84361305a7d7f6ea) , build the examples with _grunt dev_, select the _slide-pane_ example, press _open slidepane_ and make the window smaller or use the mobile device viewer in chrome dev tools you will the  see that some parts of the contents is cut out and cannot be reached.

**Expected behavior:**

The slide-pane becomes scrollable once it has more contents which does not fit in the slide-pane any more.

**Actual behavior:**

Content which does not fit into the slide-pane is cut out and cannot be reached.

### PR DESCRIPTION
**Pull request by SimonScholz**
**Type:** bug / feature

The following has been addressed in the PR:

* [ x ] There is a related issue
* [ x ] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

Not sure whether Unit or Functional tests are necessary for this PR, since I just changed the CSS.

**Description:**

Resolves #641

### CODE REVIEW

File: src/slide-pane/styles/slide-pane.m.css [hunk 1]
```diff
@@ -11,6 +11,13 @@
 	box-sizing: border-box;
 }
 
+.contentFixed {
+	overflow-y: auto;
+	position: fixed;
```
Comment (tomdye) [inline src/slide-pane/styles/slide-pane.m.css]: I believe this should just be the `overflow-y` style, we wouldn't want to apply `position: fixed` or a set margin-top here

Comment (SimonScholz) [reply src/slide-pane/styles/slide-pane.m.css]: Ok just using height 100% did the trick as well. But without it, it does not work.

Comment (SimonScholz) [reply src/slide-pane/styles/slide-pane.m.css]: So I removed position:fixed; and margin-top with my lastest commit

**Comment by SimonScholz** [comment]
Unfortunately this change breaks the slide pane tests, it seems to me that the harness.expect now also needs the new _fixedCss.contentFixed_, right?
Can you tell me how to properly fix this?

**Comment by tomdye** [comment]
Morning @SimonScholz , I've checked out your code to take a look and I noticed that the `height: 100%` addition appears to push the content off of the bottom of the page. I believe that the correct way to deal with this is to use `display: flex;` as this will allow the content to take up the full height and adapt to any layout changes.

To implement this you'll need to change the `paneFixed` and `contentFixed` classes to:

```css
.paneFixed {
	position: fixed;
	box-sizing: border-box;
	display: flex;
	flex-direction: column;
}

.contentFixed {
	overflow-y: auto;
	flex-grow: 1;
}
```

To fix the issue you are having with your tests, you'll need to add your new `fixedCss.contentFixed` class to the `slide-pane` tests [here](https://github.com/dojo/widgets/blob/master/src/slide-pane/tests/unit/SlidePane.ts#L78) and anywhere else that you see the `css.content` class being used within that test.

Hope this unblocks you, please let me know here or on Discord if you need any more assistance. Thanks again for contributing. 👍

**Comment by SimonScholz** [comment]
> Morning @SimonScholz , I've checked out your code to take a look and I noticed that the `height: 100%` addition appears to push the content off of the bottom of the page. I believe that the correct way to deal with this is to use `display: flex;` as this will allow the content to take up the full height and adapt to any layout changes.
> 
> To implement this you'll need to change the `paneFixed` and `contentFixed` classes to:
> 
> ```css
> .paneFixed {
> 	position: fixed;
> 	box-sizing: border-box;
> 	display: flex;
> 	flex-direction: column;
> }
> 
> .contentFixed {
> 	overflow-y: auto;
> 	flex-grow: 1;
> }
> ```
> 

This fixes the scroll problems. Thanks!

> To fix the issue you are having with your tests, you'll need to add your new `fixedCss.contentFixed` class to the `slide-pane` tests [here](https://github.com/dojo/widgets/blob/master/src/slide-pane/tests/unit/SlidePane.ts#L78) and anywhere else that you see the `css.content` class being used within that test.
> 

Thanks a lot for the pointer. Tests have been fixed by that.

> Hope this unblocks you, please let me know here or on Discord if you need any more assistance.
> Thanks again for contributing. +1

It's a pleasure to help and being a part of the community. Open Source rocks ;-)

**Comment by tomdye** [comment]
Thanks @SimonScholz

**Pull request merged**